src/aiovodafone/api.py

```diff
@@ -81,36 +93,56 @@ async def get_device_type(
             DeviceType:
             If the device is a Technicolor, it returns `DeviceType.TECHNICOLOR`.
             If the device is a Sercomm,     it returns `DeviceType.SERCOMM`.
+            If the device is a UltraHub,    it returns `DeviceType.ULTRAHUB`.
```

Comment:
I think we should return also the URL that worked so we have it ready for the next steps and allows us to avoid loops